### PR TITLE
add sql file for L1 lgd_orders

### PR DESCRIPTION
**Pull request by fikrihaykal41**
first version L1 lgd_orders

### CODE REVIEW

File: source/sql/dwh/bq_view/L1/visibility/lgd_orders.sql [hunk 1]
```diff
@@ -0,0 +1,563 @@
...
+  JSON_EXTRACT_SCALAR(A.data, '$.billId') AS bill_id,
+  JSON_EXTRACT_SCALAR(A.data, '$.salesOrderId') AS sales_order_id,
+  JSON_EXTRACT_SCALAR(A.data, '$.paymentId') AS payment_id,
+  JSON_EXTRACT_SCALAR(A.data, '$.purchaseOnDeliveryId') AS purchase_on_delivery_id,
+  JSON_EXTRACT_SCALAR(A.data, '$.purchaseOrderId') AS purchase_order_id,
+  IF(REPLACE(JSON_EXTRACT_SCALAR(A.data, '$.urlPurchaseOnDeliveryPdf'), '"', '') = "", NULL, REPLACE(JSON_EXTRACT_SCALAR(A.data, '$.urlPurchaseOnDeliveryPdf'), '"', '')) AS url_purchase_on_delivery_pdf,
+  JSON_EXTRACT_SCALAR(A.data, '$.orderMethod') AS order_method,
+  JSON_EXTRACT_SCALAR(A.data, '$.paymentMethodLabel') AS payment_method_label,
+  JSON_EXTRACT_SCALAR(A.data, '$.companyCategory') AS company_category,
+  JSON_EXTRACT_SCALAR(A.data, '$.urlSalesOrderPdf') AS url_sales_order_pdf,
```
Comment (toileto) [inline source/sql/dwh/bq_view/L1/visibility/lgd_orders.sql:489]: Mungkin bisa di `NULL` - in aja kalau dia kosong

File: source/sql/dwh/bq_view/L1/visibility/lgd_orders.sql [hunk 2]
```diff
@@ -0,0 +1,563 @@
...
+      JSON_EXTRACT_SCALAR(list_address, '$.zipCode') AS zip_code,
+      JSON_EXTRACT_SCALAR(list_address, '$.zipCodeId') AS zipcode_id,
+      JSON_EXTRACT_SCALAR(list_address, '$.address') AS address,
+      JSON_EXTRACT_SCALAR(list_address, '$.mainAddress') AS main_address,
+      IF(JSON_EXTRACT_SCALAR(list_address, '$.externalId') = "", NULL, JSON_EXTRACT_SCALAR(list_address, '$.externalId')) AS external_id,
+      JSON_EXTRACT_SCALAR(list_address, '$.lat') AS lat,
+      JSON_EXTRACT_SCALAR(list_address, '$.long') AS long,
+      CAST(JSON_EXTRACT_SCALAR(list_address, '$.isFulfillmentProcess') AS BOOL) AS is_fulfillment_process,
+      JSON_EXTRACT_SCALAR(list_address, '$.addressId') AS address_id,
+      IF(JSON_EXTRACT_SCALAR(list_address, '$.addressMark') = "", NULL, JSON_EXTRACT_SCALAR(list_address, '$.addressMark')) AS address_mark
```
Comment (toileto) [inline source/sql/dwh/bq_view/L1/visibility/lgd_orders.sql:76]: Ini kayanya harus gini deh

```sql

IF(
    JSON_EXTRACT_SCALAR(list_address, '$.addressMark') IN ("", "\"\""), 
    NULL,
    JSON_EXTRACT_SCALAR(list_address, '$.addressMark')
)

```

File: source/sql/dwh/bq_view/L1/visibility/lgd_orders.sql [hunk 3]
```diff
@@ -0,0 +1,563 @@
...
+      JSON_EXTRACT_SCALAR(data, '$.store.mainAddress.zipCode') AS zip_code,
+      JSON_EXTRACT_SCALAR(data, '$.store.mainAddress.zipCodeId') AS zipcode_id,
+      JSON_EXTRACT_SCALAR(data, '$.store.mainAddress.address') AS address,
+      JSON_EXTRACT_SCALAR(data, '$.store.mainAddress.mainAddress') AS main_address,
+      IF(JSON_EXTRACT_SCALAR(data, '$.store.mainAddress.externalId') = "", NULL, JSON_EXTRACT_SCALAR(data, '$.store.mainAddress.externalId')) AS external_id,
+      JSON_EXTRACT_SCALAR(data, '$.store.mainAddress.lat') AS lat,
+      JSON_EXTRACT_SCALAR(data, '$.store.mainAddress.long') AS long,
+      CAST(JSON_EXTRACT_SCALAR(data, '$.store.mainAddress.isFulfillmentProcess') AS BOOL) AS is_fulfillment_process,
+      JSON_EXTRACT_SCALAR(data, '$.store.mainAddress.addressId') AS address_id,
+      IF(JSON_EXTRACT_SCALAR(data, '$.store.mainAddress.addressMark') = "", NULL, JSON_EXTRACT(data, '$.store.mainAddress.addressMark')) AS address_mark
```
Comment (toileto) [inline source/sql/dwh/bq_view/L1/visibility/lgd_orders.sql:120]: Ini juga sama

```sql
IF(JSON_EXTRACT_SCALAR(data, '$.store.mainAddress.addressMark') IN ("", "\"\""), NULL, JSON_EXTRACT(data, '$.store.mainAddress.addressMark'))
```

**Review by toileto — CHANGES_REQUESTED** [review]
Check comment ya @fikrihaykal41 🙇‍♂️

**Review by fikrihaykal41 — COMMENTED** [review]
sudah ku revisi mas

**Review by ameliaksalwa — APPROVED** [review]
superr sekalii menurut hasil review aku sudah sesuai semua, approve

**Review by toileto — CHANGES_REQUESTED** [review]
Good job mas @fikrihaykal41! 



Love how the code is structured, it's clean!

Anyway, sebelum di release, there are several improvements ya sedikit soal query ini.

**Review by fikrihaykal41 — COMMENTED** [review]
sudah ku revisi ya mas

**Review by toileto — APPROVED** [review]
Cakep

**Pull request merged**